### PR TITLE
client: add unlock_fh_pos when abort

### PR DESCRIPTION
**Pull request by renhwztetecs**
Signed-off-by: huanwen ren <ren.huanwen@zte.com.cn>

### CODE REVIEW

**Comment by renhwztetecs** [comment]
retest this please

File: src/client/Client.cc [hunk 1]
```diff
@@ -8432,7 +8432,6 @@ int Client::_read(Fh *f, int64_t offset, uint64_t size, bufferlist *bl)
 
   if ((f->mode & CEPH_FILE_MODE_RD) == 0)
     return -EBADF;
-  //bool lazy = f->mode == CEPH_FILE_MODE_LAZY;
```
Comment (batrick) [inline src/client/Client.cc]: Please don't unnecessarily change lines.

File: src/client/Client.cc [hunk 2]
```diff
@@ -8845,7 +8854,6 @@ int Client::_write(Fh *f, int64_t offset, uint64_t size, const char *buf,
     unlock_fh_pos(f);
   }
 
-  //bool lazy = f->mode == CEPH_FILE_MODE_LAZY;
```
Comment (batrick) [inline src/client/Client.cc]: Same here.

File: src/client/Client.cc (hunk 1 above)
Comment (renhwztetecs) [inline src/client/Client.cc]: @batrick thanks for your reviews. "lazy" not in use, drop it by the way

**Comment by batrick** [comment]
`Reviewed-by: Patrick Donnelly <pdonnell@redhat.com>`

**Pull request merged**